source/sfp/1_water/sfp_114_nl_waterstaalname_plas/06_benodigdheden.Rmd

```diff
@@ -0,0 +1,113 @@
+# Benodigdheden
+
+Tabel 1 geeft een overzicht van de benodigde apparatuur en materiaal voor het nemen van een waterstaal.
+Onder 7.1, 7.2 en 7.3 worden een aantal specifieke benodigdheden van verdere uitleg voorzien.
+
++:------------------------------------------------------------+:--------------------------------------------------------------------------+
+| -   staalnameformulieren & protocol of digitaal alternatief | -   staalname-emmer                                                       |
++-------------------------------------------------------------+---------------------------------------------------------------------------+
+| -   klembord, potlood en slijper                            | -   maatbeker                                                             |
++-------------------------------------------------------------+---------------------------------------------------------------------------+
+| -   laarzen, lieslaarzen, waadpak (plaatsafhankelijk)       | -   spuiten en filters                                                    |
++-------------------------------------------------------------+---------------------------------------------------------------------------+
+| -   latex handschoenen (poederloos)                         | -   koelbox met koelelementen                                             |
++-------------------------------------------------------------+---------------------------------------------------------------------------+
+| -   plooimeter                                              | -   reserverecipiënten en hersluitbare zakjes                             |
++-------------------------------------------------------------+---------------------------------------------------------------------------+
+| -   fototoestel                                             | -   documentatie van het staalnamepunt om dit exact te kunnen lokaliseren |
++-------------------------------------------------------------+---------------------------------------------------------------------------+
+| -   recipiënten                                             | -   hamer om een gat in het ijs te maken                                  |
++-------------------------------------------------------------+---------------------------------------------------------------------------+
+
+Tabel 1: Checklist benodigd veldmateriaal.
+
+## Apparatuur
+
+Niet van toepassing
+
+## Materiaal
+
+**Staalname-emmer**
+
+Witte emmer (polyethyleen), inhoud 10 l, met maatmarkering en giettuit (figuur 1).
+
+**Maatbeker**
+
+Doorzichtige maatbeker (polypropyleen), inhoud 2 l, met maatmarkering, giettuit en handvat (figuur 1).
+
+![Figuur 1: Maatbeker (links) en staalname-emmer (rechts).](./media/image1.jpg){width="6.299660979877515in" height="4.722222222222222in"}
```

Comment:
"Figuur 1" mag geschrapt worden; anders wordt er in het gecompileerde document 'Figuur 7.1: Figuur 1:' getoond

Reply:
Ok, ik pas dit dan ook voor andere figuren aan.

Reply:
Opgelost in [51f8889](https://github.com/inbo/protocolsource/pull/107/commits/51f88891956e891ed7faac1753549da76079e8ad)